scripts/skimNtuple.py

```diff
@@ -78,6 +78,7 @@ def parseInputFileList (fileName) :
     parser.add_option ('--ttHToNonBB',       dest='ttHToNonBB', help='if it is a ttHToNonBB sample'         , default=False)
     parser.add_option ('--hhNLO',            dest='hhNLO'     , help='if it is an HH NLO sample'            , default=False,  action = 'store_true')
     parser.add_option ('--doSyst',           dest='doSyst'    , help='compute up/down values of outputs'    , default=False,  action = 'store_true')
+    parser.add_option ('--whichSgnHp',       dest='whichSgnHp', help='HH/ZH/ZZ are supported'               , default='HH')
```

Comment:
these should be added also in `scripts/skimNtuple_mib.py`

Reply:
Yes, I was waiting for the approval of the work up to know before modifying the other scripts